cmd/pace/main.go

```diff
@@ -77,3 +80,46 @@ func CreateConfigDirAndFileIfNotExists() {
 		))
 	}
 }
+
+func CreateLastSeenCommand() {
+	lastSeenCommandFilepath := path.Join(common.ConfigPath(), common.DefaultLastSeenFilename)
+	now := time.Now()
+	if content, err := os.ReadFile(lastSeenCommandFilepath); err != nil {
+		updateLastSeen(lastSeenCommandFilepath, now)
+	} else {
+		ts, err := strconv.ParseInt(strings.Trim(string(content), "\n"), 10, 64)
+		if err != nil {
+			os.Remove(lastSeenCommandFilepath)
+			updateLastSeen(lastSeenCommandFilepath, now)
+		}
+		lastSeen := time.Unix(ts, 0)
+		if lastSeen.Unix() < now.Add(-24*time.Hour).Unix() {
+			updateLastSeen(lastSeenCommandFilepath, now)
+		}
+	}
+}
+
+func updateLastSeen(lastSeenCommandFilepath string, now time.Time) {
+	CliExit(os.WriteFile(
+		lastSeenCommandFilepath,
+		[]byte(fmt.Sprintf("%d", now.Unix())),
+		0644,
+	))
+	printWelcomeMessage()
+}
+
+func printWelcomeMessage() {
+	asciiArt := `
+---------------------------------------------------------------------
+                        ____   _    ____ _____                         
+                       |  _ \ / \  / ___| ____|
+                       | |_) / _ \| |   |  _|  
+                       |  __/ ___ \ |___| |___ 
+                       |_| /_/   \_\____|_____|
+---------------------------------------------------------------------
+Hey there, cool you're using PACE!
```

Comment:
Not sure whether a full blown ASCII art is the way to go. Wouldn't this be annoying every 24 hours? We could make an opt out with a setting?